Header header
uint32 frame_number # Vicon Frame Number
vicon_mocap/Marker[] markers # Array of Markers